api/test/gateways/Jigsaw/FetchNotes.test.js

```diff
@@ -8,14 +8,23 @@ describe('JigsawFetchNotesGateway', () => {
   const jigsawEnv = '_test';
   let getSystemId;
   let doGetRequest;
+  let buildNote;
+
+  const createGateway = (
+    records,
+    existsInSystem,
+    throwsError,
+    returnsNotes
```

Comment:
what is returnsNotes?

Reply:
we used it instead of this, forgot to delete: 
<img width="566" alt="image" src="https://user-images.githubusercontent.com/54268893/74229896-34674580-4cbb-11ea-83d5-290daf9d46c8.png">